metadata/demographic/metro_micro_statistical_areas_schema.md

```diff
@@ -0,0 +1,57 @@
+# Title
+
+Metro Micro Statistical Areas
+
+## ID
+
+4e8eaf1f-eda3-47bb-974f-f99be23847b3
+
+### namelsad
+
+Current name and the translated legal/statistical description for metropolitan statistical area/micropolitan statistical area
+
+### lsad
+
+Current legal/statistical area description code for metropolitan statistical area/micropolitan statistical area.
+
+Values:
+
+- M1 = Metropolitan Statistical Area
+- M2 = Micropolitan Statistical Area
+
+### memi
+
+Current metropolitan/micropolitan status indicator.
+
+Values:
+
+- 1 = Metropolitan
+- 2 = Micropolitan
+
+### mtfcc
+
+MAF/TIGER feature class code.
+
+### funcstat
```

Comment:
This appears to be a functional status code referenced here, so I'll let you summarize it: https://www.census.gov/library/reference/code-lists/functional-status-codes.html#:~:text=The%20functional%20status%20(FUNCSTAT)%20code,gazetteer%20files%2C%20and%20other%20products.